### DISCUSSION TITLE
Android tablet?

### DISCUSSION THREAD

**Discussion by jonathanjacob** [tesla-android/issue-tracker#7] [General]
I have a spare Lenovo tab 4 8 android tablet. It works with the carlinkit app/CarPlay. Can I use the Linux Pi with this rather than  two raspberry pis? 

Thanks for the great work!

**Comment by mikegapinski** [comment]
That is not possible, Android on the Pi is modified to allow the virtual touchscreen and custom resolution matching the Model 3 browser
